app/importers/daily_calls/twilio/call_record.rb

```diff
@@ -0,0 +1,44 @@
+module DailyCalls
+  module Twilio
+    class CallRecord
+      MINIMUM_CALL_TIME = 10
+
+      class << self
+        def build(calls)
+          grouped_calls = calls.sort_by(&:start_time).group_by { |record| record.parent_call_sid || record.sid }
+
+          grouped_calls.values.map do |call_pairs|
+            call_pairs.count == 2 ? new(call_pairs) : log(call_pairs)
```

Comment:
This has led you into a slightly odd API whereby you're now relying on `log` returning `nil` for your mapped result to be the right shape. It might be cleaner to split this into two distinct operations - one to select the `call_pairs` with a `count` of `2` and another containing the rest. You can then pass the 'rest' to log and the ones meeting the criteria to something that `new`s.